modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -33,28 +33,28 @@ def index
 
         def decide
           proc_id = form_attributes['procId']
+          ptcpnt_id = form_attributes['ptcpntId']
+          decision = normalize(form_attributes['decision'])
 
-          unless proc_id
-            raise ::Common::Exceptions::ParameterMissing.new('procId',
-                                                             detail: 'procId is required')
-          end
-
-          decision = form_attributes['decision']
-
-          unless decision && %w[accepted declined].include?(normalize(decision))
-            raise ::Common::Exceptions::ParameterMissing.new(
-              'decision',
-              detail: 'decision is required and must be either "ACCEPTED" or "DECLINED"'
-            )
-          end
+          validate_decide_params!(proc_id:, decision:)
 
           service = ManageRepresentativeService.new(external_uid: 'power_of_attorney_request_uid',
                                                     external_key: 'power_of_attorney_request_key')
 
+          poa_request = validate_ptcpnt_id!(ptcpnt_id:, proc_id:, service:) if decision == 'declined'
+
+          if poa_request
+            first_name = poa_request['claimantFirstName'] || poa_request['vetFirstName'].presence ||
+                         poa_request['VSOUserFirstName'].presence
```

Comment:
cf. https://github.com/department-of-veterans-affairs/bgs-catalog/blob/main/VDC/ManageRepresentativeService/ManageRepresentativePortBinding/readPOARequestByPtcpntId/response.xml

Going from most to least specific.

Reply:
`VSOUserFirstName` is the Representative I believe..... not the Veteran / Claimant. I believe we want to remove that part.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/19362/commits/d13c04213b6b5f7fc14a2e66f3dcf8147ef42859